### ISSUE TITLE
How do you select a device?

### ISSUE THREAD

**Issue by tripplehelix** [randomouscrap98/ardugotools#19]
Managed to find 
```
Arguments:
  [<device>]    The system device to write to (use 'any' for first)
```
in `ardugotools flashcart write --help`. However, how would I select from multiple devices?

**Comment by randomouscrap98** [comment]
If you use `ardugotools device scan` it will give you a list of connected devices. One of the fields (idr which one) indicates the port the device is on. If you're on Windows, it'll be something like "COM5", and on Linux it's like "/tty/acm0" or along those lines; use that in place of "any". I'm surprised "any" doesn't work for you; do you need to flash multiple devices? If so, you can parse the output of `ardugotools device scan` and loop over them, but be warned that on Windows, the device can "hop" across COM ports depending on the state of the device.

If you need more help, let me know

**Comment by tripplehelix** [comment]
`any` works fine, I was just wondering how to select a single device. I had tried all combinations of `/dev/ttyACM0` to specify device, but for some reason it didn't work. I guess I didn't try it without quotes because now it's working.

Might I suggest explaining that `any` works in place of a device specifier in the readme. To make first use simple.

Then which information you need to then feed into `ardugotools * *` as currently it only mentions `any`. Would be nice to explain what `[<device>]` is with an example. Simply mentioning the `device` is `"Port":` would suffice. Using `[<device>]` makes me think you need to wrap `/dev/ttyACM0` in `<>`.

E.G.

```
Usage: ardugotools flashcart write [Port] [flags]

Write full flashcart to arduboy

Arguments:
  [Port]    The system device to write to (use 'any' for autodetection of a single device)
```

**Comment by randomouscrap98** [comment]
Ah, thank you for the suggestions. I went ahead and updated the readme with some examples. As for the help, it's auto-generated and I'm not sure how much I can alter it without simply disabling it and writing my own (which I'm not too keen to do; I've written enough documentation to know how quickly it gets out of sync lol). I'll look into it and see if I can't add more to the output, or tweak it somehow. I figured most people would just run the commands without specifying a device and find out most of them still work without it lol (I added that to the readme too).
